content/en/serverless/step_functions/distributed-maps.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Tracing Distributed Map states
+---
+
+This page explains our support for tracing your Map states in AWS Step Functions set to [Distributed mode][1].
+
+{{< img src="serverless/step_functions/distributed_map.png" alt="A visualization of a Step Function with a Distributed Map state." style="width:100%;" >}}
+
+## Trace Merging
+To ensure that the child workflows correctly parent the Distributed map, you must enable the "Export Map state results to Amazon S3" option. Learn more about [exporting to S3][2].
```

Comment:
I'm not quite sure what is meant by "child workflows correctly parent the Distributed map"... can we change this to:

```suggestion
To ensure that child workflows are correctly linked to their parents, enable the **Export Map state results to Amazon S3** option for your Step Function. For more information, see [Exporting to Amazon S3][2].
```

Is this what you meant?